chain/client/src/test_utils/test_loop/sync_jobs_actor.rs

```diff
@@ -0,0 +1,34 @@
+use crate::client_actions::SyncJobsSenderForClientMessage;
+use crate::sync_jobs_actor::{SyncJobsActor, SyncJobsSenderForSyncJobsMessage};
+use near_async::messaging::Handler;
+use near_async::test_loop::event_handler::LoopEventHandler;
+use near_async::test_loop::futures::TestLoopFutureSpawner;
+
+pub fn forward_messages_from_client_to_sync_jobs_actor(
+    future_spawner: TestLoopFutureSpawner,
+) -> LoopEventHandler<SyncJobsActor, SyncJobsSenderForClientMessage> {
+    LoopEventHandler::new_simple(move |msg, sync_jobs_actor: &mut SyncJobsActor| match msg {
+        SyncJobsSenderForClientMessage::_apply_state_parts(msg) => {
+            sync_jobs_actor.handle(msg);
+        }
+        SyncJobsSenderForClientMessage::_block_catch_up(msg) => {
+            sync_jobs_actor.handle(msg);
+        }
+        SyncJobsSenderForClientMessage::_resharding(msg) => {
+            sync_jobs_actor.handle_resharding_request(msg, &future_spawner);
```

Comment:
Can not call the generic handle function here as we need to pass the future_spawner. Will tackle this separately.